lib/mindwendel_web/templates/error/error_page.html.eex

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0' >
+    <title>mindwendel</title>
+    <link phx-track-static rel="stylesheet" href="<%= Routes.static_path(@conn, "/css/app.css") %>"/>
+    <script defer phx-track-static type="text/javascript" src="<%= Routes.static_path(@conn, "/js/app.js") %>"></script>
+  </head>
+
+  <body class="bg-light">
+    <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-white border-bottom mb-2">
+      <div class="container">
```

Comment:
we should probably share the navigation with the other layout as a partial, otherwise you have to add new menu links multiple times. Alternative: Personally, I don't need the menu bar at the top. I am fine with the link below the error.

Reply:
I will follow the second approach because integrating the menu bar adds a new dependency and a potential point of failure. When the menu bar breaks then the error pages will not work as well.

Reply:
Done